xml/System.Windows.Input/InputEventArgs.xml

```diff
@@ -126,9 +126,9 @@
       </ReturnValue>
       <Docs>
         <summary>Gets the time when this event occurred.</summary>
-        <value>The timestamp.</value>
+        <value>The milliseconds since last reboot. After about 24.9 days this value reaches Int32.MaxValue and restarts at (the negative) Int32.MinValue.</value>
```

Comment:
- Better: The milliseconds since last --> The number of milliseconds that have elapsed since the last 
- Int32.MaxValue can be a link: \<xref:System.Int32.MaxValue?displayProperty=nameWithType>
- Int32.MinValue can be a link: \<xref:System.Int32.MinValue?displayProperty=nameWithType>